### PR TITLE
Updated search-v2-operator-bundle publish makefile

### PR DESCRIPTION
**Pull request by dislbenn**
Signed-off-by: dislbenn <dbennett@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-release-2.7.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ tests:
         export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
         export SELF="make"
         export OSCI_PUBLISH_DELAY="0"
-        make -f Makefile.prow osci/publish
+        make -f /opt/build-harness/Makefile.prow osci/publish
```
Comment (jlpadilla) [inline ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-release-2.7.yaml:61]: This should work, but for consistency, our other projects have this in Makefile.prow

https://github.com/stolostron/search-v2-api/blob/main/Makefile.prow#L3

Comment (jlpadilla) [reply ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-release-2.7.yaml:61]: I also noticed that the project is missing `Makefile.prow` it only has `Makefile`

Comment (dislbenn) [reply ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-release-2.7.yaml]: Updated the reference to point to `Makefile.prow`.

**Comment by dislbenn** [comment]
Closing this PR in favor of https://github.com/openshift/release/pull/31918

**Pull request closed without merging**